client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -486,20 +492,25 @@ class EmailProvidersComparison extends React.Component {
 		};
 
 		return (
-			<PromoCard
-				isPrimary
-				title={ translate( 'Get your own @%(domainName)s email address', translateArgs ) }
-				image={ image }
-				className="email-providers-comparison__action-panel"
-			>
-				<p>
-					{ translate(
-						'Pick one of our flexible options to connect your domain with email ' +
-							'and start getting emails @%(domainName)s today.',
-						translateArgs
-					) }
-				</p>
-			</PromoCard>
+			<Fragment>
```

Comment:
You could use the [short fragment syntax](https://reactjs.org/docs/fragments.html#short-syntax) here and you wouldn't need to import `Fragment` i.e. `<></>`

Reply:
Thank you!